amundsen_application/static/js/components/common/SearchBar/constants.ts

```diff
@@ -4,3 +4,5 @@ export const PLACEHOLDER_DEFAULT = 'search for data resources...';
 export const BUTTON_CLOSE_TEXT = 'Close';
 
 export const SIZE_SMALL = 'small';
+
+export const INVALID_SYNTAX_MESSAGE = 'Your search termc contains invalid syntax';
```

Comment:
typo `termc`

Reply:
Maybe we can explain what characters are invalid?

```
Your search term contains invalid syntax or characters.
Alphanumeric characters [a-z] [0 - 9] . 
```

Not sure what the exact messaging here should be.

Reply:
Thanks, I leaned away from listing of which characters are valid in this can because every other special character is currently valid for the search term. I can update to specify `:` is invalid.